ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/dialog.stories.ts

```diff
@@ -180,7 +180,7 @@ class DialogWithFormComponent {
             </fudis-body-text>
           </fudis-grid>
           <hr />
-          <fudis-grid [columns]="{ xs: 1, sm: 2, md: 3 }" [marginTop]="'sm'" [marginBottom]="'sm'">
+          <fudis-grid [columns]="{ xs: 1, sm: 2, md: 3 }" [classes]="['fudis-mt-sm fudis-mb-sm']">
```

Comment:
Change above `<hr />` element to `<fudis-hr />`.
Grid columns' xs is always one so let's remove `xs: 1 `part from the JSON object.